README.md

```diff
@@ -33,11 +33,11 @@ Note that all user settings (language, keyboard layout, display mode) will persi
 **Raspberry Pi 4 Model B:** display must be connected to HDMI port closest to power jack.
 
 To set up a blank SD card with NOOBS:
-- Format an SD card as FAT32 (see instructions on how to do this below). The size of the SD card depends on which operating system(s) you want to install. 
-For a full Raspbian installation, a 16GB card is considered the minimum at the time of writing. To maximise the amount of space available on your SD card, consider using NOOBS-Lite with an internet connecton.
+- Format an SD card as FAT. See the instructions given below.
+  - Your SD card will need to be at least 16GB for Full Raspbian, or at least 8GB for all other installs.
+  - Consider using NOOBS-Lite with an internet connection if you want to leave more space for operating systems.
```

Comment:
I have copied the text from raspberrypi/documentation/installation/noobs.md, just adding the hint about noobs-lite (and fixing my typo!)